scss/_patterns_card.scss

```diff
@@ -0,0 +1,29 @@
+// Card patterns
+@mixin vf-p-card {
+
+  .p-card {
+    background: $color-light;
+    border-radius: 2px;
+    border: 1px solid $color-mid-grey;
+    padding: 1rem;
+    text-align: center;
```

Comment:
I dont think we should dictate text alignment here. The author of the page can use the util.